_projects/gitmate_testing.md

```diff
@@ -0,0 +1,43 @@
+---
+collaborating_projects:
+  - GitMate
+desc: "GitMate.io is now an MVP with proper unit tests. However, we still lack support for quick integration tests on feature additions and bug fixes. Kudos, if you could help us with that."
+developers_involved: []
+difficulty: medium
+initiatives:
+  - GSoC
+  - RGSoC
+issues:
+  - "https://gitlab.com/gitmate/open-source/gitmate-2/issues/245"
+  - "https://gitlab.com/gitmate/open-source/gitmate-2/issues/294"
+markdown: gitmate_testing.md
+mentors:
+  - nkprince007
+  - sils
+  - fneu
+name: "Improve testing on GitMate"
+requirements:
+  - "The participant should have at least one patch to [GitMate](https://gitlab.com/gitmate/open-source/gitmate-2) accepted."
+  - "The participant should demonstrate that he can run unit tests from the
+  shell in his proposal."
+  - "The participant should have sufficient knowledge in dealing with py.test
+  and Django."
+tags:
+  - GitMate
+  - py.test
+  - Django
+---
+
+[GitMate.io](https://gitmate.io/) is an automation tool for developers on their
+GitHub and GitLab repositories.
+
+The goals of this project are:
+
+- Add integration testing to GitMate for testing on live repositories.
+- Allow tests to be run in parallel in all possible combinations.
+(i.e run the plugin tests in any order).
+- (Optional) Speeding up and improving the current testing architecture.
```

Comment:
IMO improving the testing architecture is the most important point of this project, in a proposal I'd like to see concrete suggestions on how we can make it super easy for people to write integration tests for gitmate and maybe also simplify unittests so we don't have all that mocking all the time